src/values/AbstractObjectValue.js

```diff
@@ -410,11 +410,21 @@ export default class AbstractObjectValue extends AbstractValue {
         AbstractValue.reportIntrospectionError(this, P);
         throw new FatalError();
       }
+      // Extract the first existing descriptor to get its existing attributes as defaults.
+      let firstExistingDesc;
+      for (let cv of elements) {
+        invariant(cv instanceof ObjectValue);
+        firstExistingDesc = cv.$GetOwnProperty(P);
+        if (firstExistingDesc) {
+          break;
```

Comment:
Looking at the first descriptor should be strictly more correct than looking none. But to be entirely correct, shouldn't we look at all descriptors and fail if they have different (and significant, i.e. not overriden by `Desc`)  attributes?

Reply:
The second loop already asserts that they're equivalent.